clang/include/clang/CConv/ConstraintVariables.h

```diff
@@ -215,8 +216,8 @@ class PointerVariableConstraint : public ConstraintVariable {
   // the values used as arguments.
   std::set<ConstraintVariable *> argumentConstraints;
   // Get solution for the atom of a pointer.
-  const ConstAtom*getSolution(const Atom *A,
-                                  EnvironmentMap &E) const;
+  const ConstAtom* getSolution(const Atom *A,
+                               EnvironmentMap &E) const;
```

Comment:
`*` should be after the type i.e., `ConstAtom *`

Reply:
OK